backend/src/main/java/wooteco/retrospective/domain/member/Member.java

```diff
@@ -24,9 +24,7 @@ private void validateMember(String name) {
             throw new IllegalArgumentException();
         }
 
-        name = name.replaceAll(" ", "");
-
-        if (name.length() <= 0 || name.length() > MAX_LENGTH) {
+        if (name.length() <= 0 || name.length() > MAX_LENGTH || name.contains(" ")) {
```

Comment:
```suggestion
        if (name.length() <= 0 || name.length() > MAX_LENGTH || StringUtils.containsWhitespace(name)) {
```

StringUtils는 국민 라이브러리라고 합니다! 이것저것 사용해 보는건 어떨까요?